### PR TITLE
Adding cron stuff

### PR DESCRIPTION
**Pull request by elliothursh**
Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
> Links to bug tickets or user stories.

QA Plan
-------
> Fill in scenarios below. Review [QA Best Practices](https://source.sportngin.com/page/show/1227795-qa-best-practices).

### CODE REVIEW

File: recipes/_crons.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+node[:opsworks_dbreload][:databases].each do |db_name, db_info|
+  cron "#{db_name} reload" do
+    minute node[:opsworks_dbreload][:cron][:minute]
+    hour node[:opsworks_dbreload][:cron][:hour]
+    day node[:opsworks_dbreload][:cron][:day]
+    month node[:opsworks_dbreload][:cron][:month]
+    weekday node[:opsworks_dbreload][:cron][:weekday]
+    user user
+    command condition ? "#{condition} && #{basedir}/#{db_name}/dbreload" : "#{basedir}/#{db_name}/dbreload"
```
Comment (anfleene) [inline recipes/_crons.rb]: I think you should build the command outside of this resource. Inline ternaries are gross when used like this.

File: recipes/_user.rb [hunk 2]
```diff
@@ -29,13 +29,13 @@
...
-  mode 0600
+  mode '0600'
   content private_key
 end
 
 file "#{home_dir}/.ssh/authorized_keys" do
   owner user
   group group
-  mode 0700
+  mode '0700'
```
Comment (anfleene) [inline recipes/_user.rb]: This should be 600 i think

**Comment by elliothursh** [comment]
@anfleene I threw the ternary into a lib, and changed the permissions of the authorized keys. I'm going to merge this cool?

**Comment by anfleene** [comment]
:+1: Looks good :shipit:

**Pull request merged**